tutorials-v5/time-evolution/012_floquet_formalism.md

```diff
@@ -136,127 +135,45 @@ $$ \phi_\alpha(t + T) = \phi_\alpha(t) $$
 
 Hence it is enough to evaluate the modes at times $t \in [0,T]$. From these modes we can extrapolate the system state $\psi(t)$ for any time $t$. 
 
-The function `floquet_modes_table` allows to calculate the Floquet modes for multiple times in the first period. 
-
+The function `FloquetBasis` allows one to calculate the Floquet mode propagators for multiple times in the first period by specifying a list of times to `precompute`:
 
 ```python
-tlist_period = np.linspace(0, T, 50)
-fmodes_table = floquet_modes_table(f_modes0, f_energies, tlist_period, H, T)
+precompute_tlist = np.linspace(0, T, 50)
```

Comment:
Here I believe it would be better to just call it `tlist` and explain it as if you give it to the `FloquetBasis` at creation, it will be faster when used.
`FloquetBasis` works perfectly fine without it and it should not be seen as a constrain.
In v4, it always computed the propagator at least twice and by using the closest one when no exact match was available, it caused artifact in the outputs. So `FloquetBasis`is more precise and as fast as it was even without proper use of the precompute feature.

Also, this tlist can be longer that one period, this can be useful when the steps of `tlist` are not a multiple of the period.

Reply:
I changed to just `tlist`.

Yes, the v5 version is much nicer. :)